functions/create_meeting_test.ts

```diff
@@ -0,0 +1,80 @@
+import * as mf from "mock-fetch/mod.ts";
```

Comment:
just curious,
should we create a separate test folder ?
or is there a reason the test file is put in the functions folder?

Reply:
Updated to use a separate test folder